llvm/lib/Transforms/Utils/Local.cpp

```diff
@@ -3648,6 +3648,33 @@ DIExpression *llvm::getExpressionForConstant(DIBuilder &DIB, const Constant &C,
   return nullptr;
 }
 
+void llvm::remapDebugVariable(ValueToValueMapTy &Mapping, Instruction *Inst) {
+  auto RemapDebugOperands = [&Mapping](auto *DV, auto Set) {
+    for (auto *Op : Set)
+      if (auto *InstOp = Op) {
```

Comment:
You still have the `if` statements, but they're not needed either. Please can you remove those?

Reply:
Same again below - I don't think `if (Instruction *Addr = dyn_cast<Instruction>(DA->getAddress())) {` is needed on line 3661.

Reply:
@OCHyams Reworked the patch to remove those `if`s. Thanks.